src/BaGet.Core/ServiceIndex/BaGetServiceIndex.cs

```diff
@@ -6,10 +6,15 @@
 
 namespace BaGet.Core
 {
-    public class BaGetServiceIndex : IServiceIndexService
+    /// <summary>
+    /// The NuGet Service Index service, used to discover other resources.
+    /// 
+    /// See https://docs.microsoft.com/en-us/nuget/api/service-index
+    /// </summary>
+    public class BaGetServiceIndex
     {
         private readonly IUrlGenerator _url;
-
+        
```

Comment:
Please remove this whitespace

Reply:
please recheck, I think it aligns with current convention